### DISCUSSION TITLE
Other support/donation options?

### DISCUSSION THREAD

**Discussion by MichalSvatos** [florisboard/florisboard#2074] [Q&A] [answered]
Hi, I've just start using **Florisboard** and I wanted to say that I'm impress. Even if just a beta. I ditched GBoard and couldn't find a suitable replacement. I tried **OpenKeyboard** which was fine, but really basic. And then **AnySoftKeyboard**. Which was better, but somehow not working for me. 

Anyway, my question - **Is ther a plan to offer some other ways how to support this project?** I mean something like Monero or Bitcoin.

Thanks.

**Comment by patrickgold** [comment, marked as answer]
Hey,

no sorry - atm I do not intend to add any additional ways of donations for this project, as the current setup works out pretty well for me. However as always any kind of contribution is welcome, so even if you only take part in discussions or give feedback about features then this is already very much appreciated :)

**Reply by MichalSvatos** [reply]
That's a pity, but thank you for your answer. I'll be following you closely in case you change your mind 😆

Once again - **great work**!
